README.md

```diff
@@ -1,116 +1,194 @@
+# Arrays & Sorting
 
-## Table of Contents
+Problems and solutions for Arrays & Sorting session on January 17, 2020.
 
-* [About the Workshop](#about)
-* [Interview Preparation Resources](#resources)
-  * [Question Banks](#resources-qs)
-    * [Depth First (Topic-Intensive)](#resources-qs-depth)
-    * [Breadth First (General Guides, Structured Courses](#resources-qs-breadth)
-  * [Mock Interview Resources](#resources-mock)
-* [Resume Resources](#resume)
+### Table of Contents
 
-<a name="about"/>
+* [Problems](#problems)
+  * [1](#p1)
+  * [2](#p2)
+  * [3](#p3)
+* [Solutions](#solutions)
+  * [1](#s1)
+  * [2](#s2)
+  * [3](#s3)
 
-## About the Workshop
+<!-- Don't remove -->
+<a name="problems"/>
 
-Every quarter, UWB ACM hosts a 6-8 week workshop series covering common 
-technical topics and programming paradigms that are of special interest 
-to future employers. The workshop is formatted to give students an opportunity 
-to practice problem-solving techniques in a low-stress, low-risk environment 
-prior to encountering similar problems in stressful interview settings. 
+## Problems
 
-In particular, the workshop focuses on solving programming problems and 
-communicating thought processes to participants' peers. It is lead by 
-current UW Bothell students.
+<a name="p1"/>
 
-* 2019-2020 Hosts: [Lizzy](https://github.com/etcadinfinitum),
-[Kevin](https://github.com/mkhsu)
-* 2018-2019 Hosts: [Thomas](https://github.com/spacekatt), 
-[Brady](https://github.com/bpas247), [Lizzy](https://github.com/etcadinfinitum)
+### 1. PROBLEM 1 TODO :bug:
 
-<a name="resources"/>
+Source: TODO :bug:
 
-## Interview Preparation Resources
+#### Scenario
 
-<a name="resources-qs"/>
+Problem Statement TODO :bug:
 
-### Question Banks
+#### Example Input
 
-<a name="resources-qs-depth"/>
+If the problem is simple enough, remove this section. TODO :bug:
 
-#### Depth First (Topic-Intensive)
+#### Function Signature
 
-If you need intensive, concept-specific practice, the following resources 
-are excellent. Each of them includes an extensive question bank organized 
-by concept, accompanied by complete solutions.
+TODO :bug:
 
-These resources include solutions for the given problems; some are 
-published and provably correct and others are sourced from community 
-contributors.
+<!-- Don't remove -->
+Go to [Solution](#s1)   [Top](#top)
 
-* [CTCI](http://www.crackingthecodinginterview.com/), the workshop's namesake
-* [LeetCode](https://leetcode.com/)
-* [HackerRank](https://www.hackerrank.com/)
-* [GeeksForGeeks](https://www.geeksforgeeks.org/)
-* [EDUCBA Question Bank](https://www.educba.com/category/software-development/software-development-blog/top-interview-question/)[\*](#footnotes):
+<!-- Don't remove -->
+<a name="p2"/>
 
-<a name="resources-qs-breadth"/>
+### 2. Search in Rotated Sorted Array :bug:
 
-#### Breadth First (General Guides, Structured Courses)
+Source: <a href="https://www.youtube.com/watch?v=5BI0Rdm9Yhk">reference</a> :bug:
```

Comment:
Please include the LeetCode link; see writeups from other quarters for examples.